doc/PEDSnet_DQA_Documentation.md

```diff
@@ -43,19 +43,37 @@ Execute [DDL_RxNorm_Tables.sql](../Level1/library/DDL_RxNorm_Tables.sql)
 
 ### Apply DQA Checks
 
-``` R
+
+```R
 setwd("/path/to/repo/")
-source('Run_DQA.R')
+source("Run_DQA.R")
+```
+
+
+#### Run All Level 1 DQA Checks
+``` R
 runDQA(level=1)
 ```
 
-###### Level 2 For Postgres Users
+#### Run Single Level 1 DQA Check
```

Comment:
@eceowl all looking good. A couple of suggestions 
-  `Run Single Level 1 DQA Check` should be ` Run Level 1 DQA Check for a table` 
- Also instead of having two subheaders for running all vs running single checks, it would be better to move both single check commands at the end of the read me file under a `For testing only` type of header.